lib/matplotlib/transforms.py

```diff
@@ -2724,22 +2724,67 @@ def get_matrix(self):
 class ScaledTranslation(Affine2DBase):
     """
     A transformation that translates by *xt* and *yt*, after *xt* and *yt*
-    have been transformad by the given transform *scale_trans*.
+    have been transformed by the given transform *scale_trans*.
+
+    Parameters
+    ----------
+    xt, yt : number or tuple
+        If a single number, then the value of the translation *before* scaling.
+        If a tuple, then the first element is a `BboxBase` and the second
+        element is a string indicating which attribute to use as the
+        translation.
```

Comment:
It may be my addled brain at such a late hour, but perhaps provide an example string value that could elucidate this?